.travis.yml

```diff
@@ -54,8 +54,8 @@ jobs:
     - RUN_DOCKER=yes
   - stage: test
     env:
-    - JOBNAME="Linux/x86-64 without snapshot Build & Correctness Tests"
-    - OPTS="no-snapshot"
```

Comment:
No-snapshot is required, too.

Reply:
There is a duplicated one of `no-snapshot`. Diff shows the duplicated one is replaced by the new jerryx job.